.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Interrupt",
+    "build": {
+        "dockerfile": "../Dockerfile"
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "esbenp.prettier-vscode",
+                "DavidAnson.vscode-markdownlint"
+            ]
+        }
+    },
+    "postAttachCommand": ". /venv/bin/activate && bundle exec jekyll serve --host 0.0.0.0 -D --trace --future",
```

Comment:
Can we just have it use `entrypoint.sh` so there aren't two copies of this command?

Reply:
ah yep lemme test, good idea

Reply:
yep it's good! ty sir